sv_pipeline/exome/load_data.py

```diff
@@ -296,19 +326,24 @@ def format_sv(sv):
     :param sv: parsed SV
     :return: none
     """
-    sv[TRANSCRIPTS_FIELD] = [{'gene_id': gene} for gene in sv[GENES_FIELD]]
-    sv['transcriptConsequenceTerms'] = ['gCNV_{}'.format(sv[CALL_FIELD])]
+    sv[TRANSCRIPTS_CONSEQUENCE_FIELD] = {'gCNV_{}'.format(sv[CALL_FIELD])}
     if sv[SF_FIELD]:
         sv['sn'] = int(sv[SC_FIELD] / sv[SF_FIELD])
     sv['pos'] = sv[START_COL]
     sv['xpos'] = CHROM_TO_XPOS_OFFSET[sv[CHROM_FIELD]] + sv[START_COL]
     sv['xstart'] = sv['xpos']
     sv['xstop'] = CHROM_TO_XPOS_OFFSET[sv[CHROM_FIELD]] + sv[END_COL]
     sv['samples'] = []
+    sv['samples_new_call'] = []
+
+    gene_consequences = {}
     for genotype in sv[GENOTYPES_FIELD]:
         sample_id = genotype['sample_id']
         sv['samples'].append(sample_id)
 
+        if genotype[NEW_CALL_FIELD]:
+            sv['samples_new_call'].append(sample_id)
```

Comment:
A constant for `samples_new_call` is better.

Reply:
more consistant this way with how we are handling `sv['samples']` (see the above line) but okay